### DISCUSSION TITLE
Enabling Timelapse on Startup

### DISCUSSION THREAD

**Discussion by dylane-arq** [Lillifee/raspiCam#24] [Q&A] [answered]
Hey @Lillifee 

First off, amazing work! Lots of impressive things about this project, including the UI and the capability for photo, video, and stream!

I was just wondering, is it possible to have the timelapse running by default when the app starts up? I'm trying to run mine as a long-term timelapse and it would be great to not have to start it manually every restart.

**Comment by Lillifee** [comment]
Hey dylane,

Thanks a lot for your comment. I would suggest continuing the last operation after startup. I'm at the hospital, but I have a vacation over Christmas. I hope I can find some time to work on it!

**Comment by Lillifee** [comment, marked as answer]
Hey @dylane-arq 

I created a new release with a "Capture on startup" setting. It starts the capture on startup with the latest settings.
https://github.com/Lillifee/raspiCam/releases/tag/v1.3.2

![image](https://user-images.githubusercontent.com/23337626/210423657-5e9f9b62-eb77-428b-ab3c-3fc5fd889e74.png)

Let me know if this solution works as you expected.
